.env.sample

```diff
@@ -1,17 +1,23 @@
 DATABASE_URL=postgresql://user:password@localhost:5432/dbname
 SENTRY_DSN=
-HOME_URL=https://example.com/  # optional; if present, the home page will redirect to this URL
 SECRET_KEY=secret_key  # generate a secret key with `openssl rand -base64 32`
+HOME_URL=https://example.com/  # optional; if present, the home page will redirect to this URL
+
 LOG_LEVEL=info
 LOG_QUERY=false
+
 BEHIND_PROXY=false
-LISTEN_PORT=3000
```

Comment:
@dahlia this is where that `LISTEN_PORT` came from! I knew it was something that was already there and I didn't invent it in #114, turns out it should've been `PORT` in the sample environment file.